common/lib/xmodule/xmodule/js/src/video/05_video_quality_control.js

```diff
@@ -10,18 +10,14 @@ function () {
         '<button class="control quality-control is-hidden" aria-disabled="false" title="',
             gettext('High Definition'),
         '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
+            '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
+            '<span class="sr text-translation">',
+                gettext('High Definition'),
+            '</span>&nbsp;',
+            '<span class="sr control-text">',
                 // Translator note:
-                // HD stands for high definition
-                '<span class="sr text-translation">',
-                    gettext('High Definition'),
-                '</span>&nbsp;',
-                '<span class="text control-text">',
-                    // Translator note:
-                    // Values are 'off' or 'on' depending on the state of the HD control
-                    gettext('off'),
-                '</span>',
+                // Values are 'off' or 'on' depending on the state of the HD control
+                gettext('off'),
```

Comment:
@alisan617 explained to me that HD doesn't appear until you hit play. I tried it out, and now I understand about the "off" sr text.